Dockerfile

```diff
@@ -0,0 +1,101 @@
+# Base image: Apache, PHP on Debian:jessie
```

Comment:
Is this true? It looks like we are only using ubuntu:xenial base image and are installing everything else.